proto/src/determined/experiment/v1/searcher.proto

```diff
@@ -26,6 +26,8 @@ message ValidationCompleted {
   string request_id = 1;
   // value of the validation metric used to direct the search
   double metric = 2;
+  // Need the length from ValidateAfterOperation.
```

Comment:
I was thinking about how this comment could be improved, and that prompted me to ask @liamcli the question in Slack about whether we would rather just include the length in the message and not the whole operation. Including the operation feels like we are annoying the user with implementation details. Let's see what his answer is.